src/github/branch.test.ts

```diff
@@ -27,6 +27,7 @@ describe("Branch Webhook", () => {
 		await Installation.create({
 			clientKey,
 			sharedSecret: "shared-secret",
+			safeSharedSecret: "shared-secret",
```

Comment:
shouldn't the `encryptedSharedSecret` be created automatically using hooks? it'll remove the need to duplicate work.